### ISSUE TITLE
Plugin does not work

### ISSUE THREAD

**Issue by quantitative-technologies** [ashb/tmux-resurrect-virtualenvwrapper#2]
Unfortunately, this plugin does not do anything as far as I could tell.

**Comment by ghshephard** [comment]
You have to chmod + x the plugin for it work properly.  I've got it working in my tmux environment.

**Comment by Nolaan** [comment]
Still doesnt work. Are you sure it's compatible with oh my tmux for instance.
Is it possible to refine the install documentation, for instance 

"""
Add this line to the bottom of .tmux.conf:

`run-shell ~/clone/path/resurrect-venvs.tmux`
"""
Is not precise enough or at least in my case, this triggers error 126 with tpm.

**Comment by ghshephard** [comment]
Yeah - I couldn't get that to work.  It ended up with me spending about 4-6 hours and slowly dissecting the execution path and environment variable setting before I really understood how it worked.  I couldn't get run-shell to work either - but I figured chmodding +x would do the trick - and it really did.  Presumably if your other plugins (like ressurrect) are working, then the restore-venv should as well.  Definitely does for me.
